api/src/main/java/org/pmiops/workbench/workspaces/resources/UserRecentResourceServiceImpl.java

```diff
@@ -269,11 +260,11 @@ public List<DbUserRecentlyModifiedResource> findAllRecentlyModifiedResourcesByUs
   }
 
   /**
-   * Check number of entries in user_recent_resource for user, If it exceeds USER_ENTRY_COUNT,
-   * delete the one with earliest lastAccessTime
+   * Check number of entries in user_recently_modified_resource for user, If it exceeds
+   * USER_ENTRY_COUNT, delete the one with earliest lastAccessTime
    */
   private void handleUserLimit(long userId) {
-    long count = userRecentResourceDao.countUserRecentResourceByUserId(userId);
+    long count = userRecentlyModifiedResourceDao.countByUserId(userId);
```

Comment:
this method handles both old and new

Reply:
Is this because both entities get mapped to the same dao?